2016/01_january_minutes.md

```diff
@@ -0,0 +1,83 @@
+# Minutes
+
+## Attendence
+* Vance Lucas
+* Ryan Hoegg
+* Lucas Watson
+* Jeremy Green
+* Devin Clark
+* Matthew Reily
+* Jesse Harlin
+* Amanda Harlin
+* Ada Harlin
+* Mike Slack
+
+Software governance policy discussion.
+
+Jesse calls for a vote to approve software policy and techlahoma.org, us.techlahoma.org (blog roll, etc), donate.techlahoma.org.
+
+Ryan moved to not approve the software policy and to revise immediately after the meeting. Vance seconded. The vote was unanimously approved by all present board members.
+
+New tulsa data science meetup. Luke Crouch. They will be meeting at 36 North. 
+
+Jesse would like a list of all the tulsa user groups.
+
+More discussion about new groups.
+
+Jesse is going to make an issue with to get a full list of user groups. 
+
+Ryan mentioned a future discussion about "techlahoma groups" and "techlahoma endorsed groups".
+
+### Financials
+* Xero Integration updates -- Vance needs the bank password. Create an account for each user group in techlahoma.
+
+Donator obligations - review donor obligations draft and approve.
+
+Vance wants to create a sponsor obligations committee. Lucas and Kristin have volunteered for it. We will get an update from them at the next board meeting.
+
+Luke would also like to be a part of the committee.
+
+### Earmarked Donations
+We are going to have to address earmarks in the future.
+
+Jesse would like to propose there is not a reason to not take a regional earmark. Ex. OKC vs Tulsa.
+
+We need to make a policy with the critera for earmarks. 
+
+Luke proposes the sponsor committee write a draft for this. 
+Lucas proposes we handle this on a case-by-case basis. Ryan is concerned about approval time. 
+Jesse suggests we combine these ideas. 
+
+Mike would like to be involved in these discussions.
+
+### Project Updates
+This is the first month we are sponsoring code for okc.
+
+Luke: Code across hack night. Mostly an FYI
+
+Thunder Plains Committee - need members. Need to make decisions soon on venue. We will decide on chair for committee once the committe is formed.
+
+Luke: View Source Event asks if Techlahoma would be interested in an Oklahoma event. Looking for potential organizers. Not completely sure if regional events will happen.
+
+### Space Program
+We voted in December to bring Tommy Yi and John Kerber as advisors on the space program. Jesse wants to plan a meeting dedicated to this
+
+VOTE: Authorized in principal to enter into a lease. Need to identify a board member Tommy can consult with on the lease negotiations. 404 LLC will donate all their assets to Techlahoma. 404 will cease operations once the co-working.
+
+Vote was unamiously approved by all present board members.
```

Comment:
Note: I did not vote to approve this. I abstained because I don't understand exactly what "Authorized in principal" means as opposed to "Authorized"